src/jobs/test-nut.yml

```diff
@@ -10,31 +10,42 @@ parameters:
   node_version:
     description: version of node to run tests against
     type: string
-    default: "latest"
+    default: 'latest'
```

Comment:
is changing all the `"` to `'` ok? Seems like it could cause issues, but I'm not a circle expert

Reply:
yaml don't care.

I've seen these a few times...because this isn't a js/ts project, we don't have eslint in it, so different editors are formatting on their own whim.